akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphBackedFlowSpec.scala

```diff
@@ -141,7 +141,7 @@ class GraphFlowSpec extends AkkaSpec {
       "work with a Sink when having KeyedSource inside" in {
         val probe = TestSubscriber.manualProbe[Int]()
 
-        val source = Source.apply(Source.subscriber[Int]) { implicit b ⇒
+        val source = Source(Source.subscriber[Int]) { implicit b ⇒
```

Comment:
Yeah this is example I found and went the wrong way - will simplify it to Source.subscriber[Int]